tests/test_lint_recipe.py

```diff
@@ -838,17 +839,16 @@ def test_single_space_pins(self):
         lints, hints = linter.lintify(meta)
         filtered_lints = [lint for lint in lints if lint.startswith("``requirements: ")]
         expected_messages = [
-            "``requirements: build: pip   19`` should only contain a single space "
-            "between the name and the pin, i.e. ``pip 19``",
-            "``requirements: host: python >= 2`` should only contain a single space "
-            "between the name and the pin, i.e. ``python >=2``",
-            "``requirements: run: xonsh>1.0`` must contain a single space between the "
+            "``requirements: host: python >= 2`` should not contain a space between "
+            "relational operator and the version, i.e. ``python >=2``",
+            "``requirements: run: xonsh>1.0`` must contain a space between the "
             "name and the pin, i.e. ``xonsh >1.0``",
-            "``requirements: run: conda= 4.*`` must contain a single space between the "
+            "``requirements: run: conda= 4.*`` must contain a space between the "
             "name and the pin, i.e. ``conda =4.*``",
-            "``requirements: run: conda-smithy<=54.*`` must contain a single space "
+            "``requirements: run: conda-smithy<=54.*`` must contain a space "
             "between the name and the pin, i.e. ``conda-smithy <=54.*``",
         ]
+        print(filtered_lints)
```

Comment:
This probably isn't strictly needed, but it is in the tests

Reply:
so not a big deal

Reply:
Let me fix that.